packages/salesforcedx-sobjects-faux-generator/src/generator/configUtil.ts

```diff
@@ -5,52 +5,30 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {
-  ConfigAggregator,
-  ConfigFile,
-  ConfigValue,
-  StateAggregator
-} from '@salesforce/core';
-import * as path from 'path';
+import { ConfigAggregator, OrgConfigProperties } from '@salesforce/core';
 
-const defaultUserNameKey = 'defaultusername';
+async function getConfigAggregator(
```

Comment:
❤️ this being wrapped in a util

Reply:
I wonder if it should go in utils and be used everywhere we get a configAggregator

Reply:
Follow up WI: [W-11528403](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE0000130l0YYAQ/view)